examples/connectivity/plot_canica_resting_state.py

```diff
@@ -46,11 +46,10 @@
 import nibabel
```

Comment:
I guess this unneeded nibabel import may be in other examples, would you mind double-checking?